assets/scss/_component-examples.scss

```diff
@@ -396,6 +396,18 @@
   }
 }
 
+// Shadows
+
+.bd-example-shadow-utils {
+  [class^="box-shadow"] {
```

Comment:
Why this and not just a class?

Reply:
It was because I was using classes with prefix .box-shadow like .box-shadow-top without mandatory .box-shadow class together. Sometime after, I changed it and I forced the use of .box-shadow always, like `.box-shadow.box-shadow-top`, but forgot to change it. Now, I did the change.

Thanks!

Reply:
@gijsbotje suggested to not force the use of .box-shadow class (that was an question that I was in doubt). So, now, I added the `[class^="box-shadow"] ` again and removed the usage of class `.box-shadow`